src/config.h

```diff
@@ -0,0 +1,50 @@
+#ifndef CONFIG_H_PIBRI8ND
+#define CONFIG_H_PIBRI8ND
```

Comment:
_PIBRI8ND is random string to be "unique" but i will rename it in way you suggest